WcaOnRails/app/views/teams/edit.html.erb

```diff
@@ -6,38 +6,37 @@
   <%= horizontal_simple_form_for @team, html: { class: 'are-you-sure no-submit-on-enter' } do |f| %>
     <%= f.input :friendly_id, label: "Friendly ID" %>
     <%= f.input :name %>
-    <%= f.input :description %>
 
-      <div class="team-members-wrapper">
-        <h3>Members</h3>
-        <%= wca_table table_class: "team-members", hover: false, striped: false do %>
-          <thead>
-            <tr>
-              <th>User</th>
-              <th>Start date</th>
-              <th>End date</th>
-              <th>Team leader</th>
-              <th></th> <!-- Extra column for .table-greedy-last-column -->
-            </tr>
-          </thead>
-          <tbody>
-          <% # We need to use `sort_by` instead of `order` to keep the new builded, unsaved objects.
-             team_members = @team.team_members.sort_by do |m|
-               # We call `to_s` on the dates to avoid an exception when one of them is nil.
-               [(m.end_date.nil? ? 1 : 0), m.end_date.to_s, m.start_date.to_s]
-             end.reverse!
-          %>
-          <%= f.simple_fields_for :team_members, team_members do |t| %>
-            <%= render "team_member_fields", f: t %>
-          <% end %>
-          </tbody>
+    <div class="team-members-wrapper">
+      <h3>Members</h3>
+      <%= wca_table table_class: "team-members", hover: false, striped: false do %>
+        <thead>
+          <tr>
+            <th>User</th>
+            <th>Start date</th>
+            <th>End date</th>
+            <th>Team leader</th>
+            <th></th> <!-- Extra column for .table-greedy-last-column -->
+          </tr>
+        </thead>
+        <tbody>
+        <% # We need to use `sort_by` instead of `order` to keep the new builded, unsaved objects.
+           team_members = @team.team_members.sort_by do |m|
```

Comment:
Urg, getting punished for code I didn't write here (I was just fixing indentation).

Fixing this is somewhat non-trivial (there are multiple controller actions that lead to here), so I'd rather address this in a separate PR.